src/web/utils.ts

```diff
@@ -18,3 +18,36 @@ export const degToRad = (degrees: number) => (degrees * Math.PI) / 180;
 
 export const coneToDeviation = (degrees: number) =>
   Math.cos(degToRad(degrees / 2));
+
+export function calculateViewScale(view: HTMLElement) {
+  const styles = getComputedStyle(view);
+
+  const resultScales = {
+    scaleX: 1,
+    scaleY: 1,
+  };
+
+  const scales = styles.scale.split(' ');
+
+  if (scales[0] !== 'none') {
+    resultScales.scaleX *= parseFloat(scales[0]);
+  }
+
+  if (scales[1]) {
+    resultScales.scaleY *= parseFloat(scales[1]);
+  }
```

Comment:
Why is the `0` element checked with `!== 'none'` while the `1` element is checked with a boolean conversion?

Also, if you're multiplying numbers by 1, wouldn't it be better to set these values instead of multiplying them? 
`resultScales.scaleX *= parseFloat(scales[0])` should be equivalent to `resultScales.scaleX = parseFloat(scales[0])`, since as far as i see, `resultScales.scaleX` is always `1`

Reply:
Good catch! Done in [0474578](https://github.com/software-mansion/react-native-gesture-handler/pull/2943/commits/0474578f94034f04e14f7fc9c88b2223de989674).